apps/music/js/ui/views/player_view.js

```diff
@@ -763,27 +763,14 @@ var PlayerView = {
       this.audio.currentTime = Math.floor(seekTime);
     }
 
-    var startTime = this.audio.startTime;
```

Comment:
I could swear it used to exist. Maybe it was in a draft spec and then removed? It was supposed to be for things like live streams. If you started streaming at 15:02, then this gave you a way to display that number as the start point. But it does appear completely undocumented now.